### ISSUE TITLE
dead raises ValueError in case a file is deleted (since it remains listed with "git ls-files")

### ISSUE THREAD

**Issue by bagerard** [asottile/dead#29]
Hi,
Just hit this twice in a week. I'm using `dead` to clean up a large codebase and realized that as soon as you delete a file, `dead` starts complaining that it can't find that same file in subsequent run
```
Traceback (most recent call last):
  File "/home/***/.pyenv/versions/3.6.5/envs/cmp/bin/dead", line 8, in <module>
    sys.exit(main())
  File "/home/***/.pyenv/versions/3.6.5/envs/cmp/lib/python3.6/site-packages/dead.py", line 302, in main
    for filename, is_test in _filenames(files_re, exclude_re, tests_re):
  File "/home/***/.pyenv/versions/3.6.5/envs/cmp/lib/python3.6/site-packages/dead.py", line 213, in _filenames
    'python' not in tags_from_path(filename)
  File "/home/***/.pyenv/versions/3.6.5/envs/cmp/lib/python3.6/site-packages/identify/identify.py", line 38, in tags_from_path
    raise ValueError('{} does not exist.'.format(path))
ValueError: myapp/lib/helpers.py does not exist.
```
I quickly check and the problem is that it relies on `git ls-files`, which lists also the deleted files.
The workaround is to commit the change, then re-execute `dead` but I thought it is worth fixing this.

I havn't seen an option in `git ls-files` to lists everything except the deleted file but I see that I can list the deleted files with `git ls-files --deleted` so I guess we can use that to subtract the deleted files.

Let me know your thoughts and if that seems like a good approach, I'd be happy to implement it.

**Comment by asottile** [comment]
the easiest is probably to add an existence check

**Comment by bagerard** [comment]
Thanks for the quick feedback, makes sense. I'll work on a PR in the next days

**Comment by asottile** [comment]
thanks again, this has been released as v1.0.0 :tada:
